File: T6. C# CONDITIONAL STATEMENTS AND LOOPS - EXERCISES/P08.CaloriesCounter.cs
using System;

namespace P08.CaloriesCounter
{
    class Program
    {
        static void Main(string[] args)
        {
            int ingredientsNumber = int.Parse(Console.ReadLine());
            int caloriesCounter = 0;
            int totalCalories = 0;


            for (int i = 0; i < ingredientsNumber; i++)
            {
                string ingredients = Console.ReadLine().ToLower();

                switch (ingredients)
                {
                    case "cheese":
                        caloriesCounter = 500;
                        break;
                    case "tomato sauce":
                        caloriesCounter = 150;
                        break;
                    case "salami":
                        caloriesCounter = 600;
                        break;
                    case "pepper":
                        caloriesCounter = 50;
                        break;
                    default:
                        caloriesCounter = 0;
                        break;
                }
                totalCalories += caloriesCounter;
            }
            Console.WriteLine($"Total calories: {totalCalories}");
        }
    }
}

File: T6. C# CONDITIONAL STATEMENTS AND LOOPS - EXERCISES (Java)/P08_CaloriesCounter.java
import java.util.Scanner;

public class P08_CaloriesCounter {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        int ingredientsNumber = Integer.parseInt(in.nextLine());
        int calories = 0;
        int totalCalories = 0;

        for (int i = 1; i <= ingredientsNumber ; i++) {

            String ingredient = in.nextLine().toLowerCase();

            switch (ingredient) {
                case "cheese":
                    calories = 500;
                    break;

                case "tomato sauce":
                    calories = 150;
                    break;

                case "salami":
                    calories = 600;
                    break;

                case "pepper":
                    calories = 50;
                    break;
                    default:
                        calories = 0;

            }
            totalCalories += calories;
        }
        System.out.printf("Total calories: %d", totalCalories);
    }
}
